go/private/actions/compilepkg.bzl

```diff
@@ -218,9 +215,8 @@ def emit_compilepkg(
             sources = sources,
             cgo_go_srcs = cgo_go_srcs_for_nogo,
             archives = archives,
+            out = out_nogo,
```

Comment:
```suggestion
            out_diagnostics = out_diagnostics,
```

Reply:
Is this resolved?

Reply:
Sorry, I forgot to push. Will send a follow-up